src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -3,15 +3,21 @@ import styled from 'styled-components';
 import { shape, string, number } from 'prop-types';
 import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
 import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { useLocation } from 'react-router-dom';
 import MetadataContainer from '../../containers/Metadata';
 import ATIAnalytics from '../../containers/ATIAnalytics';
 import ChartbeatAnalytics from '../../containers/ChartbeatAnalytics';
 import Grid, { GelPageGrid } from '#app/components/Grid';
 import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
 import OnDemandHeadingBlock from '#containers/RadioPageBlocks/Blocks/OnDemandHeading';
 import ParagraphBlock from '#containers/RadioPageBlocks/Blocks/Paragraph';
 import AudioPlayerBlock from '#containers/RadioPageBlocks/Blocks/AudioPlayer';
 import EpisodeImage from '#containers/RadioPageBlocks/Blocks/OnDemandImage';
+import LinkedData from '#containers/LinkedData';
+import getMediaId from '#lib/utilities/getMediaId';
+import getMasterbrand from '#lib/utilities/getMasterbrand';
+import getEmbedUrl from '#lib/utilities/getEmbedUrl';
```

Comment:
personally I think utility functions that are not used by any other page type should be located as close as possible to the component that uses them rather than be added to a global utilities folder

Reply:
The utility functions are used by both LiveRadio and OndemandRadio pages. I thought about creating a `utilities` folder in the `pages` folder, but felt that might not be ideal.

Reply:
The utility functions just hoist the shared logic used by both pages that was in the AudioPlayer.

Reply:
Oh ok sorry I didn't realised it was shared between those 2 page types. That's fine then 🙂